src/create.jl

```diff
@@ -124,6 +124,7 @@ RasterStack("created.nc")
 └───────────────────────────────────────────────────────────────────────────────────────────┘
 ```
 """
+create
```

Comment:
```suggestion
function create end
```
Sorry I like the explicit style, for newbies it looks weird as-is...